Makefile

```diff
@@ -141,9 +140,15 @@ clean:
 	rm -f *_stringer.go	# generated by `go generate`
 	rm -f *_mock.go		# generated by `go generate`
 
-test: bindata
+test: build
 	./test.sh
 
+# create all targets for make tab completion
```

Comment:
this is cool :)

Reply:
thx, if for one like make, good choice for this project :)